.github/workflows/release.yaml

```diff
@@ -1,48 +1,41 @@
-name: Release
-on:
-  workflow_dispatch:
+name: release
+
+on: [workflow_dispatch]
 
 jobs:
-  release:
-    name: Release
+  build:
```

Comment:
keep release job name

Reply:
I copied from this [job-name](https://github.com/osl-incubator/poc-semantic-release/blob/main/.github/workflows/release.yaml)